Recrutify/Recrutify.DataAccess/Repositories/CandidateRepository.cs

```diff
@@ -74,5 +74,28 @@ private async Task UpdateWithArrayFiltersAsync(Guid id, UpdateDefinition<Candida
             var updateOptions = new UpdateOptions { ArrayFilters = arrayFilters };
             await GetCollection().UpdateOneAsync(filter, updateDefinition, updateOptions);
         }
+
+        public Task<List<Candidate>> GetByIdsAsync(IEnumerable<Guid> candidatesId)
+        {
+            var filter = _filterBuilder.In(u => u.Id, candidatesId);
+            return GetCollection().Find(filter).ToListAsync();
+        }
+
+        public Task CreateFeedbacksByIds(IEnumerable<Guid> ids, Guid projectId, Feedback feedback)
+        {
+            var updateBuilder = Builders<Candidate>.Update;
+            var updateDefinition = updateBuilder
+                    .AddToSet("ProjectResults.$[projectResult].Feedbacks", feedback);
+            var binaryProjectId = new BsonBinaryData(projectId, GuidRepresentation.Standard);
+            var arrayFilters = new List<ArrayFilterDefinition>
+                {
+                   new BsonDocumentArrayFilterDefinition<ProjectResult>(new BsonDocument("projectResult.ProjectId", binaryProjectId)),
+                };
+
+            var filter = _filterBuilder.In(x => x.Id, ids);
```

Comment:
move filter to first line of method, to be in one style with other methods

Reply:
+